releasetasks/test/test_releasetasks.py

```diff
@@ -210,3 +212,66 @@ def test_funsize_en_US_scopes_dep_signing(self):
                 self.assertIsNone(generator["task"].get("scopes"))
                 self.assertItemsEqual(signing["task"]["scopes"], ["signing:cert:dep-signing", "signing:format:mar", "signing:format:gpg"])
                 self.assertIsNone(balrog["task"].get("scopes"))
+
+    def test_l10n_one_chunk(self):
+        graph = make_task_graph(
+            source_enabled=False,
+            updates_enabled=False,
+            l10n_platforms=["win32"],
+            enUS_platforms=["win32"],
+            branch="mozilla-beta",
+            product="firefox",
+            repo_path="releases/mozilla-beta",
+            revision="abcdef123456",
+            l10n_changesets={
+                "de": "default",
+                "en-GB": "default",
+                "zh-TW": "default",
+            },
+            l10n_chunks=1,
+        )
+
+        self._do_common_assertions(graph)
+
+        task = get_task_by_name(graph, "mozilla-beta_firefox_win32_l10n_repack_1")
```

Comment:
I addressed the former, I'm not sure what you mean about checking for _repack_0 though?